src/lexer.c

```diff
@@ -361,10 +384,17 @@ void tryParseExportStatement () {
       ch = commentWhitespace(true);
       while (true) {
         char16_t* startPos = pos;
-        readToWsOrPunctuator(ch);
+
+        ch = readToWsOrPunctuatorOrQuote(ch);
```

Comment:
I think this will only ever match the first quote for a string like `export { "p" as x } from 'z'` right? If so we may be able to replace this with just:

```suggestion
        if (!isQuote(ch)
          ch = readToWsOrPunctuator(ch);
```

The thing is, in the quote branch above we should still be scanning through the quotes (like the previous comment), otherwise we may fail on edge cases like:

```js
export { "p as 'z' from 'asdf'" as "z'" } from 'asdf';
```